app/src/main/java/org/eyeseetea/malariacare/views/question/AKeyboardQuestionView.java

```diff
@@ -43,4 +43,6 @@ protected void hideKeyboard(View view) {
     public interface onAnswerChangedListener {
         void onAnswerChanged(View view, String newValue);
     }
+
+    public abstract EditText getInputView();
```

Comment:
I think is not a good idea expose input view to outside, what ever logic with input view child should be resolved in its question view parent